base/linalg/bidiag.jl

```diff
@@ -154,19 +172,10 @@ for func in (:A_ldiv_B!, :Ac_ldiv_B!, :At_ldiv_B!)
         B
     end
 end
-for func in (:A_ldiv_Bt!, :Ac_ldiv_Bt!, :At_ldiv_Bt!)
-    @eval function ($func)(A::Union(Bidiagonal, AbstractTriangular), B::AbstractMatrix)
-        tmp = similar(B[:, 2])
-        m, n = size(B)
-        nm = n*m
-        for i = 1:size(B, 1)
-            copy!(tmp, 1, B, i:m:nm, n)
-            ($func)(A, tmp)
-            copy!(B, i:m:nm, tmp, 1, n)
-        end
-        B
-    end
-end
+
+A_ldiv_Bt(A::Union(Bidiagonal, AbstractTriangular), B::AbstractMatrix) = A_ldiv_B!(A, transpose(B))
+At_ldiv_Bt(A::Union(Bidiagonal, AbstractTriangular), B::AbstractMatrix) = At_ldiv_B!(A, transpose(B))
+Ac_ldiv_Bt(A::Union(Bidiagonal, AbstractTriangular), B::AbstractMatrix) = Ac_ldiv_B!(A, transpose(B))
```

Comment:
I think these should go into `operators.jl` without type annotations. It must be the right thing to do in general.